openghg/analyse/_scenario.py

```diff
@@ -446,6 +450,10 @@ def add_flux(
         if self.fluxes is not None:
             if not hasattr(self, "species"):
                 flux_values = list(self.fluxes.values())
```

Comment:
Just wondering why `self.fluxes.values()` would be an empty list? We probably want to make this so `self.fluxes` would just be None in this circumstance rather than still having keys but no values?